.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+on:
+  release:
+    branches:
+      - main
+    types: [created]
+name: Publish release
+jobs:
+  release_linux:
+    runs-on: ubuntu-latest
```

Comment:
"latest" is problematic, as that could change at any time and re-create the issue.

You need to pick a version which has a stable GLIBC. You can also run this inside a container:

https://github.com/ctron/hat/blob/444f4ad1f3c5cdb2e309fe50d1812676ea715ddc/.travis.yml#L19

Reply:
Changed to specific tag Ubuntu-18.04

Works on
- Centos:8
- Red hat universal base image
- Fedora 34
- Arch Linux
- Debian ( after `apt-get install libssl-dev` )

Reply:
For posterity: Debian users can install `apt-get install libssl1.1` if they don't want a `dev` dependency